### ISSUE TITLE
Current project status

### ISSUE THREAD

**Issue by DanielRuf** [Rich-Harris/butternut#200]
Hi, is this project still actively maintained?

It seems there was no activity since mid 2017.

**Comment by interhive** [comment]
Doesn't appear so. A real shame because it's a powerful tool.

**Comment by Rich-Harris** [comment]
Yep, afraid I don't have any time to work on this for the forseeable future. I'm currently using [Terser](https://github.com/terser-js/terser), which has proven reliable.

I may get back to it one day but it's less of a priority than when I started it because Terser is a good option.

**Comment by interhive** [comment]
Cool, thanks for the update @Rich-Harris
